cis/app/frontend/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "cis-front",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "browserify ./scripts/main.js -o ../static/build/bundle.js",
+    "watch": "watchify ./scripts/main.js -o ../static/build/bundle.js -v -d"
```

Comment:
Ces lignes émettent systématiquement un warning relou, mais inoffensive : 
`You did not set any plugins, parser, or stringifier. Right now, PostCSS does nothing. Pick plugins for your case on https://www.postcss.parts/ and use them in postcss.config.js.`

Reply:
comme c'est juste pour plus tard on se fera une issue follow-up sur le sujet

Reply:
https://github.com/entrepreneur-interet-general/CIS-front/issues/46